nbs/examples/Multivariate_with_TSMixer.ipynb

```diff
@@ -16,11 +16,11 @@
    "source": [
```

Comment:
<p>remove bold for MSE in TSMixer (paper) column</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Nixtla/neuralforecast/pull/938/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='34'/>

Reply:
Removed